apps/prairielearn/src/pages/instructorAssessmentAccess/instructorAssessmentAccess.types.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from 'zod';
+
+export const AssessmentAccessRulesSchema = z.object({
+  mode: z.string(),
+  uids: z.string(),
+  start_date: z.string(),
+  formatted_start_date: z.string().optional(),
+  end_date: z.string(),
+  formatted_end_date: z.string().optional(),
+  credit: z.string(),
```

Comment:
👀 this is probably a string because of the old query that casts everything to a string, but after changing that, you might consider leaving this as a number.

Reply:
The old query has been updated and this is now a number in [1dfef07](https://github.com/PrairieLearn/PrairieLearn/pull/9934/commits/1dfef07527ed0296ff3cdfa7c58d85fce2304b97).